doc/api/url.md

```diff
@@ -235,6 +235,19 @@ url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
 url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
 ```
 
+Note that `url.resolve()` method depends on the protocol given in the `from` parameter. 
+Some protocols like `https`, `file`, `ftp`, `wss` and `gopher` have special cases.
+
+For example:
+
+```js
+url.resolve('https://foo.tld', 'bar')   // 'https://foo.tld/bar'
+url.resolve('wss://foo.tld', 'bar')      // 'wss://bar'
+url.resolve('ftps://foo.tld', 'bar')     // 'ftps://bar'
```

Comment:
To follow on my previous comment, I think it would be neat to include an example with a trailing / in a URL. Like adding this example to your list:
```javascript
url.resolve("ftps://foo.tld/", "bar"); // 'ftps://foo.tld/bar'
```